tests/integration-tests/tests/ad_integration/test_ad_integration.py

```diff
@@ -460,6 +461,12 @@ def _run_user_workloads(users, test_datadir, remote_command_executor):
         user.assert_job_succeeded(job_id)
 
 
+def _check_whoami(users):
+    logging.info("Checking whoami")
+    for user in users:
+        user.run_remote_command("whoami && srun whoami")
```

Comment:
can we validate what these responses are?

Reply:
Done